ngi_reports/stockholm/project_summary.py

```diff
@@ -27,27 +27,18 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         super(Report, self).__init__(config, LOG, working_dir, **kwargs)
 
         ## Project name - collect from the command line if we have it
-        if kwargs.get('project') is not None:
-            self.project_info['ngi_name'] = kwargs['project']
-            self.project_name = self.project_info['ngi_name']
-        else:
-            ## Check to see if we have the project ID yet
-            try:
-                self.project_name = self.project_info['ngi_name']
-            except KeyError:
-                self.LOG.error("No project name found - please specify using '--project'")
-                raise KeyError("No project name found - please specify using '--project'")
+        self.project_name = kwargs.get('project', self.project_info.get('ngi_name'))
+        if not self.project_name:
+            self.LOG.error("No project name/id provided - please specify using '--project'")
+            raise SystemExit
```

Comment:
Huh, does `raise SystemExit` give no traceback? I didn't know that. If there's a reason to keep it then that's fine, leave it as it is.. I agree that the traceback doesn't really help here.